packages/flutter/lib/src/widgets/tooltip.dart

```diff
@@ -0,0 +1,920 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
I suggest renaming all files related to `RawTooltip` as `raw_tooltip`. This convention is used by other raw widgets and going against it makes it counterintuitive when I can't find anything searching for the file based on the widget name.